src/types/oidc-provider.ts

```diff
@@ -0,0 +1,15 @@
+//
+
+import type { InteractionResults, UnknownObject } from "oidc-provider";
+
+//
+
+export interface OIDCContextParams extends UnknownObject {
```

Comment:
can you confirm the export is needed here ? There is none for the other types.

Reply:
"export"s are because its a `*.ts` file.
do you prefer a `*.d.ts` ?

Reply:
My preference goes for the most coherent option regarding the existing code : `*.d.ts`

Reply:
Done in 9416aad8d1aa9fb317c6d171d150259c51313d41 but...
I would avocate against global interfaces. I understand that it's goes with the actual code flow but I think it should be changed in favor of types imports from files.